src/app/cases/merge-duplicates-case-guide/merge-duplicates-case-guide.component.html

```diff
@@ -0,0 +1,33 @@
+<div mat-dialog-title>
+  <h1>{{ 'strings.headingMergeGuide' | translate }}</h1>
+  <div mat-dialog-actions>
+    <button mat-icon-button mat-dialog-close class="button-dialog-close">
+      <mat-icon class="material-icons-round">highlight_off</mat-icon>
+    </button>
+  </div>
+</div>
+
+<div mat-dialog-content>
+  <h2>{{ 'strings.headingIntroduction' | translate }}</h2>
+  <div [innerHtml]="'strings.infoMergingExplanation' | translate" class="guide-info"></div>
+  <br />
```

Comment:
there's no need to add extra tags to create space;
you can achieve the same by adding extra margin on h2

Reply:
Done.